bin/server_docker/ubuntu/docker-entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+OS_DIR_CONFIG="/etc/mysterium-node"
+OS_DIR_DATA="/var/lib/mysterium-node"
+OS_DIR_RUN="/var/run/mysterium-node"
+
+if [ ! -d /dev/net ]; then
+    mkdir -p /dev/net
+fi
+if [ ! -c /dev/net/tun ]; then
+    mknod /dev/net/tun c 10 200
+fi
+
+if [ ! -d "$OS_DIR_RUN" ]; then
+    mkdir -p $OS_DIR_RUN
+fi
+
+exec /usr/bin/mysterium_server \
+  --config-dir=$OS_DIR_CONFIG \
+  --data-dir=$OS_DIR_DATA \
+  --runtime-dir=$OS_DIR_RUN \
+  $@
```

Comment:
What about defaults file in Debian init.d service?
`bin/client_package/installation/default`

Reply:
Made a common accepted style. Define DAEMON_OPTS empty var in config file, eval var in initd.sh script